comtypes/tools/codegenerator.py

```diff
@@ -345,6 +346,11 @@ def ArrayType(self, tp):
     _enumvalues = 0
     def EnumValue(self, tp):
         value = int(tp.value)
+        if keyword.iskeyword(tp.name):
+            # XXX use logging!
+            if __debug__:
```

Comment:
I wouldn't rely on the `__debug__` flag here - [from the docs](https://docs.python.org/3/library/constants.html#__debug__) it's True unless Python was run with `-O`. In practice, no-one I know runs Python with `-O`, so it will ~always print the message.